README.md

```diff
@@ -250,4 +361,14 @@ The automatized workflow is based on:
 [3] Andre Santos Ribeiro, Luis Miguel Lacerda, Hugo Alexandre Ferreira, Multimodal Imaging Brain Connectivity Analysis (MIBCA) toolbox, 2015 Jul 14. [Available here](https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4511822/)
 
 
-[![Build Status](https://travis-ci.org/the-virtual-brain/tvb-recon.svg?branch=master)](https://travis-ci.org/the-virtual-brain/tvb-recon) [![Coverage Status](https://coveralls.io/repos/github/the-virtual-brain/tvb-recon/badge.svg)](https://coveralls.io/github/the-virtual-brain/tvb-recon)
+[![Build Status](https://img.shields.io/badge/Build-error-critical)](https://travis-ci.org/the-virtual-brain/tvb-recon) [![Coverage Status](https://img.shields.io/badge/Coverage-unknown-informational)](https://coveralls.io/github/the-virtual-brain/tvb-recon)
+
+
+### Contributors 
+<!-- ALL-CONTRIBUTORS-LIST:START - Do not modify this section -->
+<!-- prettier-ignore -->
+| [<img src="https://avatars0.githubusercontent.com/u/674853?s=460&v=4" width="100px;"/><br /><sub><b>Lia Domide</b></sub>](https://github.com/liadomide)<br />        | [<img src="https://avatars2.githubusercontent.com/u/5189886?s=460&v=4" width="100px;"/><br /><sub><b>marmaduke woodman</b></sub>](https://github.com/maedoc/)<br /> | [<img src="https://avatars3.githubusercontent.com/u/3601158?s=400&v=4" width="100px;"/><br /><sub><b>Bogdan Neacsa</b></sub>](https://github.com/bogdanneacsa)<br />          | [<img src="https://avatars3.githubusercontent.com/u/1563810?s=400&v=4" width="100px;"/><br /><sub><b>Paula Sanz-Leon</b></sub>](https://github.com/pausz)<br /> | [<img src="https://avatars0.githubusercontent.com/u/2657768?s=400&v=4" width="100px;"/><br /><sub><b>Michael Burgstahler</b></sub>](https://github.com/twotribes)<br />    | [<img src="https://avatars3.githubusercontent.com/u/48951339?s=460&v=4" width="100px;"/><br /><sub><b>Saksham Raghuvanshi</b></sub>](https://github.com/sraghuvanshi)<br />                               | 
```

Comment:
I would not keep this section here. 
Github repo already has the list of contributors nicely rendered.

Reply:
Sure, ma'am, I'll remove this section.